operator-catalog-initialization-test.yml

```diff
@@ -8,10 +8,11 @@
     run_upstream: false
     operator_work_dir: "/home/jenkins/agent/test-operator"
     work_dir: "{{ lookup('env', 'WORKSPACE') }}"
-    offline_cataloger_bin_path: "/usr/local/bin/offline-cataloger"
     production_quay_namespace: "redhat-operators"
+    oc_bin_path: "/usr/local/bin/oc"
```

Comment:
I cannot see any documentation that `oc` has to be stored at this location
Moreover we expect some minimal version of client as well

Reply:
This playbook is meant for usage in the operator-utils container. In other usages the oc location (and version) will be set as part of the install_prereqs role.

Reply:
Fair enough.

But then we should document required expectation for local-test-operator.yml

Reply:
Added a note to the install_prereqs role.